style.css

```diff
@@ -74,6 +73,59 @@
   position: absolute;
 }
 
+.shadow {
+   box-shadow: 0 -1px 20px 0px rgba(0, 0, 0, 0.3);
+}
+
+.footer {
+  background-color: #F5C85F;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  padding: 30px;
+  padding-bottom: 50px;
```

Comment:
this propery is included inside the padding you defined in line 86. although you can do `padding: 30px 30px 50px 30px;` I would recommend you leave the top and bottom paddings the same

Reply:
in the layout picture from the Trello card the bottom margin is greater than the top one. That's why i made it this way, i prefer them to be equal too.